package.json

```diff
@@ -78,7 +78,7 @@
     "postcss-loader": "1.2.1",
     "prompt": "1.0.0",
     "react-addons-test-utils": "15.4.1",
-    "react-hot-loader": "^1.3.1",
+    "react-hot-loader": "3.0.0-beta.6",
```

Comment:
Actually latest has 3.0.0-beta.6.  I dun goofed and had `3.0.0` which isn't an actual version that can be pulled down.

If you guys would like I can PR us back(revert HMR 3.0).  LMK @coryhouse  and my apologies.